canonical/ubuntu.com
# WD-221 Set-up community docs

## Done

- Add discourse module
- Update navigation
- Remove unneeded pages
- Add redirects for the removed pages

## QA

- Open the demo
- Go to /community
- Click docs
- Check the docs work as expected
- Check the redirects
